sources/packages/api/src-sql/Restrictions/Create-initial-restrictions.sql

```diff
@@ -1,67 +1,204 @@
-INSERT INTO sims.restrictions(id,restriction_type,restriction_code,description)
-VALUES
-(1,'Federal','C4','# of weeks >= 468, input by CSLP');
-
-INSERT INTO sims.restrictions(id,restriction_type,restriction_code,description)
-VALUES
-(2,'Federal','C5','# of weeks >= 520, input by CSLP');
-
-INSERT INTO sims.restrictions(id,restriction_type,restriction_code,description)
-VALUES
-(3,'Federal','C6','# of weeks >= 288, input by CSLP');
-
-INSERT INTO sims.restrictions(id,restriction_type,restriction_code,description)
-VALUES
-(4,'Federal','C7','# of weeks >= 340, input by CSLP');
-
-INSERT INTO sims.restrictions(id,restriction_type,restriction_code,description)
-VALUES
-(5,'Federal','CA','Fed, near maximum, Type A');
-
-INSERT INTO sims.restrictions(id,restriction_type,restriction_code,description)
-VALUES
-(6,'Federal','CBO','Federal, near maximum, Type B');
-
-INSERT INTO sims.restrictions(id,restriction_type,restriction_code,description)
-VALUES
-(7,'Federal','CC','Standard claim, input by CSLP');
-
-INSERT INTO sims.restrictions(id,restriction_type,restriction_code,description)
-VALUES
-(8,'Federal','CD','Death (History Code)');
-
-INSERT INTO sims.restrictions(id,restriction_type,restriction_code,description)
-VALUES
-(9,'Federal','CI','Permanent disability, input by CSLP');
-
-INSERT INTO sims.restrictions(id,restriction_type,restriction_code,description)
-VALUES
-(10,'Federal','CO','Students has rec''d O/A-input by Province');
-
-INSERT INTO sims.restrictions(id,restriction_type,restriction_code,description)
-VALUES
-(11,'Federal','CR','Put-back (R/S Loan)rec''d/paid , Input  CSLP');
-
-INSERT INTO sims.restrictions(id,restriction_type,restriction_code,description)
-VALUES
-(12,'Federal','CRB','Bankrupcy filed, input by CSLP or Lender');
-
-INSERT INTO sims.restrictions(id,restriction_type,restriction_code,description)
-VALUES
-(13,'Federal','CRD','Death, input by CSLP or Lender');
-
-INSERT INTO sims.restrictions(id,restriction_type,restriction_code,description)
-VALUES
-(14,'Federal','CRJ','Judgement registered, input CSLP/Lender');
-
-INSERT INTO sims.restrictions(id,restriction_type,restriction_code,description)
-VALUES
-(15,'Federal','CRP','Convicted of offense re CSL, CSLP/Lender');
-
-INSERT INTO sims.restrictions(id,restriction_type,restriction_code,description)
-VALUES
-(16,'Federal','CT','Temp. claim, rec''d/not paid, Input CSLP');
-
-INSERT INTO sims.restrictions(id,restriction_type,restriction_code,description)
-VALUES
-(17,'Federal','CX','R/S Loan 90 days in arrears, Input Lender');
+INSERT INTO
```

Comment:
When we decided to do the truncate on the restrictions table I was under the impression that we would not be changing this file. Does something changed or am I under the wrong understanding?

Reply:
mm....this was done as a part of fixing the id issue, for the future env. 
![image](https://user-images.githubusercontent.com/77353155/168678777-03bbfb54-948a-4b93-a826-ba0eddf1b507.png)

Reply:
Because we decided to do the TRUNCATE it is no longer needed. That said I made another comment because the TRUNCATE was not executed.

Reply:
I believe this file is not required to be a part of PR if truncate is used.